processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/PartitionContext.java

```diff
@@ -165,4 +191,12 @@ public void close() throws Exception {
         resume();
         commitControl.close();
     }
+
+    // visible for testing
+    QuotaUsage quotaUsage(byte[] key) {
```

Comment:
I think this should be named to indicate that it actually updates usage for better understandability.